package.json

```diff
@@ -30,9 +30,10 @@
     "@fortawesome/ember-fontawesome": "https://git@github.com/mansona/ember-fontawesome.git#update-get-config",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "broccoli-funnel": "^2.0.2",
+    "ember-auto-import": "^2.0.0",
     "ember-cli-babel": "^7.26.10",
     "ember-cli-htmlbars": "^5.7.2",
-    "ember-showdown-prism": "^3.0.0"
+    "ember-showdown-shiki": "^1.2.1"
```

Comment:
isn't this creating an implicit peer on showdown? do you think we should make it explicit here too?

Reply:
It does! ([ref](https://github.com/IgnaceMaes/ember-showdown-shiki/blob/main/ember-showdown-shiki/package.json#L112-L114))

But `ember-cli-showdown` isn't. I'm guessing that somehow also was an implicit peer of `ember-showdown-prism`? As that one needed to be added explicitly to make it work.

Reply:
yea maybe 🤷 its tough to know what the right thing to do with peers like this is 🙈 do you have any preference? should we leave it as it is?

Reply:
yes but I'm more asking if we should explicitly state it as a peer here. You're right that consuming apps would need to define it explicitly 👍 but that would be true regardless if we state it as a peer at this level